pandas/core/missing.py

```diff
@@ -478,6 +486,85 @@ def _akima_interpolate(xi, yi, x, der=0, axis=0):
         return [P(x, nu) for nu in der]
 
 
+def _cubicspline_interpolate(xi, yi, x, axis=0, bc_type="not-a-knot", extrapolate=None):
```

Comment:
i don’t think we put a doc string for others
we should either link them instead

Reply:
Do you mean to remove the parameters of the docstring here that are already present in the SciPy method? I am following the previous logic of the methods `_from_derivatives` and `_akima_interpolate`. I agree that the doc for the `bc_type` parameter is quite verbose and it could be substituted by a link to the SciPy method doc. However, that same parameter (the control of boundary conditions) is what makes this function different from the simpler alternative using `scipy.interpolate.interp1d `with cubic splines and it could be worth having it explicitly in the docs. What do you think? Thanks.

Reply:
do we have doc strings for other interpolate methods that are just direct scipy calls?

Reply:
No, but as with `_from_derivatives` and `_akima_interpolate`, this is not just a direct SciPy call (see resolved conversation below with @mroeschke for details), that's why I added the doc string too.